src/components/UserActions/UserActions.ts

```diff
@@ -269,17 +268,16 @@ export class UserActions extends Component {
             }
             const viewedByCustomerElement = document.createElement('span');
             new ViewedByCustomer(viewedByCustomerElement, undefined, this.bindings, args.result);
-            args.item.querySelector('.coveo-result-row:last-child .coveo-result-cell').appendChild(viewedByCustomerElement);
+            const resultLawRow = args.item.querySelector('.coveo-result-row:last-child .coveo-result-cell');
```

Comment:
When I was talking about the selector, I meant just the string `.coveo-result-row:last-child .coveo-result-cell` :P